report_thesis/src/sections/problem_definition.tex

```diff
@@ -91,37 +69,23 @@ \subsubsection{Matrix Effects}
+\subsection{Problem Formulation}
+The objective of this research is to develop a computational model, denoted as $\mathcal{F}: \mathbb{R}^N \rightarrow \mathbb{R}^{n_o}$, to predict major oxide concentrations in geological samples from processed \gls{libs} spectral data. 
 
-Following preprocessing to correct instrumental effects and calibrate spectra, the cleaned data is input into machine learning models.
-These models, trained on databases of Earth-based and synthetic Martian analogs, output quantitative analyses of chemical compositions in weight percentages of the target oxides \cite{wiensPreflightCalibrationInitial2013, cleggRecalibrationMarsScience2017}.
-\citet{cleggRecalibrationMarsScience2017} undertook this task and created a pipeline for predicting the concentration of oxides in Martian soil samples, referred to as \gls{moc}.
+\subsubsection{Evaluation Metrics}\label{subsubsec:problem_evaluation_metrics}
+To evaluate the performance of these models, we will use the \gls{rmse} to measure accuracy and the sample standard deviation of prediction errors to assess robustness.
+We define accuracy as the ability of a model to predict the composition of major oxides in geological samples, while robustness refers to the stability of these predictions across samples.
 
-More recently, in 2022, the Perseverance rover landed on Mars, equipped with advanced instruments designed to continue the exploration and analysis of the Martian surface.
-This rover also uses a \gls{libs} instrument, called SuperCam, which is the successor to \gls{chemcam}.
-The Perseverance mission highlighted the ongoing research effort in developing elemental quantification models using \gls{libs} data \cite{andersonPostlandingMajorElement2022}, demonstrating its continued importance as a research field.
+The metric used to evaluate the accuracy of the models is the \gls{rmse}:
+\[
+\text{RMSE} = \sqrt{\frac{1}{n} \sum_{i=1}^{n} (\mathbf{v}_i - \hat{\mathbf{v}}_i)^2}
+\]
+where \( \mathbf{v}_i \) is the vector of actual oxide concentrations for the \( i \)-th sample, \( \hat{\mathbf{v}}_i \) is the corresponding vector of predicted oxide concentrations, and \( n \) is the total number of samples. 
+This measure quantifies the average magnitude of the prediction error across all predicted values.
 
-The use of \gls{libs} on the Curiosity rover within the MSL mission shows how computational advancements can enhance our understanding of extraterrestrial geology.
-By effectively quantifying chemical compositions from \gls{libs} data, we can infer the historical climatic conditions of Mars, offering clues to its past habitability.
+Robustness is evaluated using the sample standard deviation of prediction errors:
+\[
+\sigma_{error} = \sqrt{\frac{1}{n-1} \sum_{i=1}^{n} (e_i - \bar{e})^2}
+\]
+where \( e_i = \mathbf{v}_i - \hat{\mathbf{v}}_i \) and \( \bar{e} \) is the mean error.
+A lower standard deviation indicates a more robust model across different samples.
```

Comment:
I really agree with this. Our notes said to move it somewhere else and I was not entirely sure whether to keep it either exactly for this reason. 
The explanation and formulas exist in proposed approach, so it might make sense to just entirely remove from here?